config/ompi_check_libfca.m4

```diff
@@ -58,7 +55,16 @@ AC_DEFUN([OMPI_CHECK_FCA],[
 			   CPPFLAGS=$CPPFLAGS_save
 			   LDFLAGS=$LDFLAGS_save
 			   LIBS=$LIBS_save],
-			   [ompi_check_fca_happy="no"])
+			  [AC_SUBST([coll_fca_HOME], "/")
```

Comment:
I don't quite understand the purpose of this check -- is it to catch the `--with-hca` (with no value) case?

If so, it might be better (and more in line with other configure.m4 scripts, I think?) to separate the user's desire to have hcoll support vs. whether or not they supplied a directory (i.e., use two different variables).  And then only have one call to `OPAL_CHECK_PACKAGE`.